modules/ztp-deploying-user-defined-firmware-configuration-with-gitops-ztp.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-deploying-far-edge-sites.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-deploying-user-defined-firmware-configuration-with-gitops-ztp_{context}"]
+= Deploying user-defined firmware to cluster hosts with {ztp}
+
+You can deploy user-defined firmware to cluster hosts by configuring the `SiteConfig` custom resource (CR) to include a hardware profile that you want to apply during cluster host provisioning.
+You can configure hardware profiles to apply to cluster hosts in the following scenarios:
+
+* All hosts in the cluster
+* Only hosts that meet certain criteria in the cluster
+* Individual hosts in the cluster
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have installed {rh-rhacm-first} and logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have provisioned a cluster that is managed by {rh-rhacm}.
+
+* You created a Git repository where you manage your custom site configuration data.
+The repository must be accessible from the hub cluster and be defined as a source repository for the Argo CD application.
+
+.Procedure
+
+. Create the host firmware profile files that contain the firmware settings you want to apply at the site, cluster, and individual node level.
+For example:
++
+.site-firmware.profile
+[source,yaml]
+----
+BootMode: Uefi
+LogicalProc: Enabled
+ProcVirtualization: Enabled
+----
++
+.cluster-firmware.profile
+[source,yaml]
+----
+EnergyEfficientTurbo: Enabled
+ProcC1E: Disabled
+TpmSecurity: "On"
+----
++
+.node-firmware.profile
+[source,yaml]
+----
+EnergyEfficientTurbo: Disabled
+ProcTurboMode: Enabled
+ProcPwrPerf: OsDbpm
+ProcC1E: Disabled
+ProcCStates: Enabled
+----
+
+. Save the hardware profile YAML files relative to the `kustomization.yaml` file that you use to define how to provision the cluster, for example:
++
+[source,terminal]
+----
+example-ztp/install
+    └── site-install
+          ├── siteconfig-example.yaml
+          ├── kustomization.yaml
+          └── firmware-profiles
+                ├── site-firmware.profile
+                ├── cluster-firmware.profile
+                └── node-firmware.profile
```

Comment:
Same here, we only need to show one profile,

Reply:
Ah ok I was confused about that actually 😂